assets/js/modules/analytics-4/datastore/partial-data.js

```diff
@@ -558,6 +558,9 @@ const baseSelectors = {
 						reportID:
 							'analytics-4_get-partial-data-report-options_store:selector_RESOURCE_TYPE_PROPERTY',
 					};
+
+				default:
+					return undefined;
```

Comment:
Cases like this just feel a bit needless when `return undefined` is already default JS behaviour 😅